package.json

```diff
@@ -17,7 +17,7 @@
   "scripts": {
     "test": "jest",
     "storybook": "run-s build:svgs:index storybook:dev",
-    "storybook:dev": "start-storybook --port 9001 --config-dir .storybook --static-dir ./public",
+    "storybook:dev": "start-storybook --docs --port 9001 --config-dir .storybook --static-dir ./public",
```

Comment:
`--docs` starts storybook in `Documentation Mode`, which means that when you load a Storybook it defaults to opening the `Docs` tab of the first story it opens instead of the `Canvas` tab. Since this PR moves the Documentation info from the Story itself to the official `Docs` page, this effectively maintains the current behavior (otherwise the Storybook would just load a single story of a stateful component.)